src/Command/Drupal/DrupalDeployCommand.php

```diff
@@ -354,22 +368,57 @@ private function runDeployHooks(Project $project, LocalApplication $app) {
 
     $this->stdErr->writeln("<info>[*]</info> Executing deployment hooks for <info>$project->id" . '-' . $app->getId() . "</info>...");
 
-    $sh = new Shell();
+    $project_dir = $this->extCurrentProject['www_dir'] . '/' . $app->getId();
     foreach (explode("\n", $appConfig['hooks']['deploy']) as $hook) {
       if ($hook != "cd " . $app->getDocumentRoot()) {
         if (strlen($hook) > 0) {
           $this->stdErr->writeln("Running <info>$hook</info>");
           if (stripos($hook, 'updb')) {
-            $sh->executeSimple($hook, $this->extCurrentProject['www_dir'] . '/' . $app->getId());
+            $this->shellHelper->executeSimple($hook, $project_dir);
           }
           else {
-            $sh->execute(explode(' ', $hook), $this->extCurrentProject['www_dir'] . '/' . $app->getId());
+            $this->shellHelper->execute(explode(' ', $hook), $project_dir);
           }
         }
       }
     }
   }
 
+  /**
+   * Execute the post-build hooks.
+   * @param $project
+   */
+  protected function runPostBuildHooks(array $appConfig, $buildDir) {
+        $result = TRUE;
+        if (!isset($appConfig['hooks']['build'])) {
+            return null;
+        }
+        $this->output->writeln('Running post-build hooks adapted to local environment.');
```

Comment:
I've also tested this now. It works fine, as far as I can tell. 
The only thing I would ask for is to keep the messaging consistent and change the line above to something on the same pattern as this: https://github.com/britishcouncil/platformsh-cli/blob/12d49235288e9fcd99ff5875b1a446c37ac468c1/src/Command/Drupal/DrupalDeployCommand.php#L369